src/interfaces/IContactUs.ts

```diff
@@ -0,0 +1,22 @@
+//Contact Us Interface
+interface IContactUs {
+  _id?: string;
+  name: string;
+  email: string;
+  message: string;
+  createdAt: Date;
+  updatedAt: Date;
+  deletedAt?: Date | null;
+}
+
+//Contact Us State interface
+interface IContactUsStore {
+  contact: IContactUs | null;
+  contacts: IContactUs[] | null;
+  ArchiveContact: IContactUs | null;
```

Comment:
Change archive to deleted and do the same for other one. Also make sure to use simple letter as first letter.